scene/resources/box_shape_3d.h

```diff
@@ -46,6 +59,19 @@ class BoxShape3D : public Shape3D {
 	void set_extents(const Vector3 &p_extents);
 	Vector3 get_extents() const;
 
+#ifdef DEBUG_ENABLED
+	void set_use_custom_faces_color(bool p_use_custom_faces_color);
+	bool get_use_custom_faces_color() const;
+	void set_faces_texture(const Ref<Texture> &p_faces_texture);
+	Ref<Texture> get_faces_texture() const;
+	void set_faces_color(const Color &p_faces_color);
+	Color get_faces_color() const;
+	void set_faces_texture_size(float p_faces_texture_size);
+	float get_faces_texture_size() const;
```

Comment:
Shouldn't these be defined on `Shape3D` rather than repeated for every shape?

Reply:
At first I did that but not all shape type are implemented

Reply:
I could maybe add a CollisionShape3DVisual (abstract) and move everything on it ? But I'll still need to have the `ArrayMesh` generation in the child class cause' i need to know the kind of shape

Reply:
Which Shape3D types don't or can't have it implemented?